### PR TITLE
Add e2e test

### PR DESCRIPTION
**Pull request by zeroalphat**

### CODE REVIEW

**Comment by umezawatakeshi** [comment]
It seems OK for now (except Dockerfile.daemon)

File: e2e/suite_test.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+var _ = BeforeSuite(func() {
+	By("prepare pod for profiling")
+	_, err := kubectl(nil, "apply", "-f", "./manifests/profiled-pod.yaml")
+	Expect(err).NotTo(HaveOccurred())
+
+	By("waiting for profiled-pod to be ready")
+	Eventually(func() error {
+		res, err := kubectl(nil, "get", "pod", "profiled-pod", "-o", "json")
+		if err != nil {
```
Comment (morimoto-cybozu) [inline e2e/suite_test.go:38]: This is just a matter of preference.
You can use Gomega's assertions even in `Eventually()`.

```
Eventually(func(g Gomega) {
  ...
  g.Expect(err).NotTo(HaveOccurred())
  ...
}).Should(Succeed())
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: e2e/Makefile [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	@echo "Choose one of the following target"
+	@echo
+	@echo "setup      Setup tools"
+	@echo "start      Start kind cluster and install accurate"
+	@echo "test       Run e2e tests"
+	@echo "logs       Save logs as logs.tar.gz"
+	@echo "stop       Stop the kind cluster"
+
+.PHONY: setup
+setup: bin_dir kubectl kustomize kind
```
Comment (morimoto-cybozu) [inline e2e/Makefile:30]: `bin_dir` is no longer necessary.

File: e2e/Makefile [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	ln -sf $(notdir $<) $@
+
+$(KUBECTL)-$(E2ETEST_K8S_VERSION):
+	mkdir -p $(dir $@)
+	$(CURL) -o $@ https://storage.googleapis.com/kubernetes-release/release/v$(E2ETEST_K8S_VERSION)/bin/$(OS)/$(ARCH)/kubectl
+	chmod a+x $@
+
+.PHONY: bin_dir
+$(BIN_DIR):
+	mkdir -p $(BIN_DIR)
```
Comment (morimoto-cybozu) [inline e2e/Makefile:86]: ditto

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Review by umezawatakeshi — APPROVED** [review]

**Review by morimoto-cybozu — APPROVED** [review]

**Pull request merged**